tdrs-frontend/src/components/EditProfile/EditProfile.test.js

```diff
@@ -590,14 +590,16 @@ describe('EditProfile', () => {
       target: { name: 'stt', value: 'alaska' },
     })
 
-    expect(store.dispatch).toHaveBeenCalledTimes(1)
+    // There should be four dispatches, three called `onChange` during
+    // the above selections, and 1 to fetch the STT list
+    expect(store.dispatch).toHaveBeenCalledTimes(4)
 
     const form = wrapper.find('.usa-form').hostNodes()
     form.simulate('submit', {
       preventDefault: () => {},
     })
 
-    expect(store.dispatch).toHaveBeenCalledTimes(2)
+    expect(store.dispatch).toHaveBeenCalledTimes(6)
```

Comment:
These are bumped due to additional calls to `dispatch` from the `useSelector` in the new `useEventLogger` hook.